src/app/CreateRecording/CustomRecordingForm.tsx

```diff
@@ -402,16 +402,16 @@ export const CustomRecordingForm: React.FC = () => {
   return (
     <>
       <Text component={TextVariants.small}>
-        JDK Flight Recordings are compact records of events which have occurred within the target JVM. Many event types
+        JDK Flight Recordings are compact records of events which have occurred within the target JVM. Many Event Types
```

Comment:
same here about `event types`

Reply:
Here too I was trying to match the UI 
![image](https://github.com/cryostatio/cryostat-web/assets/56743004/364de281-956f-4e91-a642-cfecfbbf01ce)